spec/win_finder_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'win_finder'
+
+describe WinFinder do
+  let(:game_double) { Class.new { extend WinFinder } }
+
+  context 'when board contains a winning combination' do
```

Comment:
It's maybe a good place for parameterized testing via checking the different winning combinations.